packages/@react-stately/color/src/Color.ts

```diff
@@ -94,7 +94,7 @@ class RGBColor extends Color {
       while (values.length > 0) {
         colors.push(parseInt(values.splice(0, 2).join(''), 16));
       }
-      colors[3] = colors[3] !== undefined ? colors[3] / 255 : undefined;
+      colors[3] = colors[3] !== undefined ? colors[3] / 255 : 1;
```

Comment:
I'm looking at this line and line #100 and it looks like we are defining alpha to be a range between 1 and 255, but it looks like its 0.0 to 1.0 or 0 to 255. What I mean by "defining alpha to be" is the default is 1 not 0.

https://developer.mozilla.org/en-US/docs/Web/CSS/hex-color
https://developer.mozilla.org/en-US/docs/Web/CSS/color_value/rgba